docs/extend/plugins_volume.md

```diff
@@ -164,3 +164,52 @@ this point.
 
 Respond with a string error if an error occurred.
 
+
+### /VolumeDriver.Get
+
+**Request**:
+```json
+{
+    "Name": "volume_name"
+}
+```
+
+Get the volume info.
+
+
+**Response**:
+```json
+{
+  "Name": "volume_name",
+  "Mountpoint": "/path/to/directory/on/host",
+  "Err": ""
+}
+```
```

Comment:
Oh you're right. I'll change the doc.
As it is currently coded was intentional.
It is equivalent to a function return following this pattern:
```go
 func Get(name string) (*foo, error)
```